google/cloud/spanner/benchmarks/multiple_rows_cpu_benchmark.cc

```diff
@@ -402,20 +418,37 @@ class ReadExperiment : public Experiment {
       SimpleTimer timer;
       timer.Start();
 
-      grpc::ClientContext context;
       google::spanner::v1::ReadRequest request{};
       request.set_session(*session);
+      request.mutable_transaction()
+          ->mutable_single_use()
+          ->mutable_read_only()
+          ->Clear();
       request.set_table("KeyValue");
-      request.add_columns("Key");
-      request.add_columns("Data");
+      for (auto const& name : columns) {
+        request.add_columns(name);
+      }
       *request.mutable_key_set() = cs::internal::ToProto(key);
-      auto stream = stub->StreamingRead(context, request);
+
       int row_count = 0;
       google::spanner::v1::PartialResultSet result;
+      std::vector<google::protobuf::Value> row;
+      grpc::ClientContext context;
+      auto stream = stub->StreamingRead(context, request);
       for (bool success = stream->Read(&result); success;
            success = stream->Read(&result)) {
-        if (!result.chunked_value()) {
-          row_count += result.values_size() / 2;
+        if (result.chunked_value()) {
+          // We do not handle chunked values in the benchmark.
+          continue;
+        }
+        row.resize(columns.size());
+        std::size_t index = 0;
+        for (auto& value : *result.mutable_values()) {
+          row[index++] = std::move(value);
```

Comment:
Opt: move post ++ to pre ++ on next line?

Reply:
Good idea, fixed.